src/util.ts

```diff
@@ -26,3 +26,22 @@ export const urlWithContextAsQuery = (url: URL, context: IContext) => {
         });
     return urlWithQuery;
 };
+
+const sortObjectProperties = (
+    obj: Record<string, any>
```

Comment:
Plutôt utiliser `unknown` que `any` si possible (partout)
```suggestion
    obj: Record<string, unknown>
```